rocky/reports/views/aggregate_report.py

```diff
@@ -107,6 +110,20 @@ class ExportSetupAggregateReportView(
     current_step = 4
     report_type = AggregateOrganisationReport
 
+    def post(self, request, *args, **kwargs):
+        selected_plugins = request.POST.getlist("plugin", [])
+
+        for selected_plugin in selected_plugins:
+            try:
+                get_katalogus(self.organization.code).enable_boefje_by_id(selected_plugin)
```

Comment:
Extract a variable `client` outside of the for-clause so it can be reused
```suggestion
        client = get_katalogus(self.organization.code)
        for selected_plugin in selected_plugins:
            try:
                client.enable_boefje_by_id(selected_plugin)
```